retail-managment-system/rms-ui/src/app/domain/supplier/model/supplier.model.ts

```diff
@@ -0,0 +1,10 @@
+export interface Supplier {
```

Comment:
we did folder models not model.
to continue with same structure and naming

Reply:
ok